validphys2/src/validphys/checks.py

```diff
@@ -44,8 +44,7 @@ def check_pdf_is_montecarlo(ns, **kwargs):
 
 
 @make_argcheck
-def check_pdf_is_montecarlo_or_symmhessian(ns, **kwargs):
-    pdf = ns['pdf']
+def check_pdf_is_montecarlo_or_symmhessian(pdf, **kwargs):
     etype = pdf.error_type
```

Comment:
```suggestion
@make_check
def check_pdf_is_montecarlo_or_symmhessian(ns, **kwargs):
    pdf = ns['pdf']
    etype = pdf.error_type
```

Reply:
In reportengine it seems `make_argcheck` is preferred https://github.com/NNPDF/reportengine/blob/79eec2e33c5aab3998fd58e9c3cf84bd2c2696e7/src/reportengine/checks.py#L10

why should we use `make_check` and `ns` instead? It also a bit clearer with `arcgcheck` and `pdf` since that check is only active for PDFs in any case.

Reply:
Sure, I am ok with that.

Reply:
But what's the reason for using one over the other?

Reply:
Not sure tbh, I was suggesting it because the other functions like `check_pdf_is_montecarlo` make use of it and hence it would be more uniform, but I changed my mind.

I think you are right though that `make_check` should not be used anymore since it is weird that one can access the namespace `ns` from validphys.

I am thinking that we could change the others to be `make_argcheck` as well at this point.